### ISSUE TITLE
Lenses description not saved

### ISSUE THREAD

**Issue by mat-k-witts** [anvc/scalar#231]
Noticing a fairly persistent error when adding / removing some relationships. The description field for lenses is not being saved. I can try to identify what action is causing this if this will help.

**Comment by eloyer** [comment]
Yes, if you can come up with a reproducible case that'd be helpful, thanks.
